### DISCUSSION TITLE
Is quasar protectiong API ?

### DISCUSSION THREAD

**Discussion by OutLandishnessSad** [quasarframework/quasar#14167] [General - Components / Directives / etc]
Hey there, just wanna ask if quasar is protecting api ? :)

**Comment by MilosPaunovic** [comment]
What do you mean by it?

**Reply by OutLandishnessSad** [reply]
Lets say i have this code.. Can someone else enter it ? by url + /api/v1/users/style/update/
`app.post('/api/v1/users/style/update/', (req, res) => {

  let color = ''
  let newcolor = req.body.params.color
  console.log('Here you are in your backend')
  console.log(`This is my color from BackEND ${newcolor}`)

  color = Sessions('_', 'Style')

  if (color == newcolor) {
    // Here you will in futur update the database
    console.log('Same')
    return res.sendStatus(401)
  }
  else return res.sendStatus(200)
})`

**Reply by MilosPaunovic** [reply]
As I see, that is your backend code, which has nothing to do with Quasar itself.
